packages/host/app/components/operator-mode/interact-submode.gts

```diff
@@ -325,24 +329,27 @@ export default class InteractSubmode extends Component<Signature> {
         if (!item.card.id && updatedCard.id) {
           this.operatorModeStateService.trimItemsFromStack(item);
         } else {
-          this.operatorModeStateService.replaceItemInStack(item, {
-            ...item,
-            card: updatedCard,
-            request,
-            format: 'isolated',
-          });
+          this.operatorModeStateService.replaceItemInStack(
+            item,
+            item.clone({
+              request,
+              format: 'isolated',
+            }),
+          );
         }
       }
     }
   });
 
-  @action private edit(item: StackItem) {
-    this.operatorModeStateService.replaceItemInStack(item, {
-      ...item,
-      request: new Deferred(),
-      format: 'edit',
-    });
-  }
+  private edit = restartableTask(async (item: StackItem) => {
```

Comment:
this action didn't need to turn into a task--I'll change it back (it was a left over thought that i forgot to cleanup)